src/post.ts

```diff
@@ -0,0 +1,16 @@
+import * as core from '@actions/core';
+import { Docker } from './model';
+
+export async function run() {
+  try {
+    const runnerTemporaryPath = process.env.RUNNER_TEMP;
```

Comment:
I asked myself what's `RUNNER_TEMP`, it's documented here:  
https://docs.github.com/en/actions/learn-github-actions/environment-variables

> RUNNER_TEMP | The path to a  temporary directory on the runner. This directory is emptied at the  beginning and end of each job. Note that files will not be removed if  the runner's user account does not have permission to delete them. For  example, D:\a\_temp
> -- | --

Reply:
I've taken this from the already existing code that relies on it too. 

Ideally this would be extracted into own function, so it is evaluated the same on all the branches. 
But I didn't want to rewrite the whole docker.ts, since I didn't find any unit tests and wanted to minimize the risk of breaking something.

Reply:
You can use https://github.com/game-ci/unity-test-runner/blob/main/src/model/action.ts to `get tempPath`

Reply:
Ok, I've actually implemented something that ended up in Input, but it clearly belongs to the Action. Going to move it there.

Reply:
I did not want to refactor the command generation of docker which should be changed to use this new tempPath.

It also deserves to not be text, but rather just an array, so it can be passed to the action/exec as args and it would correctly do all the escaping of arguments.

Also it would be easy to reuse some common base for the linux and win commands (cidfile, env, ...).